cfme/rest/gen_data.py

```diff
@@ -238,7 +240,7 @@ def _finished():
 
 
 def a_provider():
```

Comment:
Yes. There is also stuff like `setup_first_provider`, `setup_single_provider`, `vmware_provider` and also just `provider`. All of that will (like I said "hopefully) come together nicely and be unified.